src/components/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { fetchCommentsByArticleId } from "../utils";
+
+export default function Comments({ article_id }) {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetchCommentsByArticleId(article_id)
+      .then((res) => {
+        return res.comments;
+      })
+      .then((comments) => {
+        setComments(comments);
+      });
+  }, []);
+
+  return (
```

Comment:
The article should be displayed while comments are loading, so we could have them load independently 🙂

Reply:
I see! Ok, will do that! :)